CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 ## New Functionality
 
 - Support custom axios options for all request builders.
+- Support disabling csrf token request as an option for all request builders.
```

Comment:
I found this [line](https://github.com/SAP/cloud-sdk-js/blob/f12daa9abfd9373240d7251609f9d41e8ff6eb90/packages/core/src/http-client/http-client.ts#L237) here. Is the change from false to true as the default value potentially a change in behavior that should be mentioned in the competability notes?